CMakeLists.txt

```diff
@@ -371,6 +380,10 @@ add_test(NAME TEST_TOPK COMMAND topk market
   ${gunrock_INCLUDE_DIRS}/simple_example/bips98_606.mtx --undirected)
 set_tests_properties(TEST_TOPK PROPERTIES FAIL_REGULAR_EXPRESSION "INCORRECT")
 
+add_test(NAME TEST_GRMAT COMMAND grmat market
+  ${gunrock_INCLUDE_DIRS}/simple_example/bips98_606.mtx --normalized --compensate --undirected)
+set_tests_properties(TEST_GRMAT PROPERTIES FAIL_REGULAR_EXPRESSION "INCORRECT")
```

Comment:
See next comment, we don't need GRMAT `ctests`, also, I don't think the app you have outputs the expression "CORRECT" or "INCORRECT", so I am not sure if this will ever fail.

Reply:
I will remove this.